packages/buffetjs-core/src/components/HeaderActions/index.js

```diff
@@ -14,20 +14,20 @@ function HeaderActions({ actions }) {
   return (
     <Wrapper>
       {actions.map((action, index) => {
-        const { Component, disabled, title, onClick } = action;
+        const { Component, disabled, label, onClick } = action;
```

Comment:
Changing the API is a breaking change but ok if you think it's really necessary.